Posts container creation

eliazardan opened the pull request:
New Pull request updated with Figma attributes

eliazardan committed:
Update PostsContatainer.tsx

New attributes for the Post Container

eliazardan committed:
Update PostsContatainer.tsx

Updated Attributes from Figma

eliazardan committed:
Merge branch 'main' into Posts-Container-Creation

Tauriqbarron commented on Components/Notice Board/PostsContatainer.tsx:
```diff
@@ -1,9 +1,26 @@
+    width: 340px;
+    height: 194px;
+    left: calc(50% - 340px/2);
+    top: 72px;
```
Dont need this.

Tauriqbarron commented on Components/Notice Board/PostsContatainer.tsx:
```diff
@@ -1,9 +1,26 @@
   return( 
-  <View>
+  <View style={styles.msgBox}>
+    <Text>"Lets see what this text looks like</Text>
```
Change this to PostContentContainer it will be available to import soon

eliazardan committed:
Update PostsContatainer.tsx

Removed height left and top attributes

Tauriqbarron commented on Components/Notice Board/PostsContatainer.tsx:
```diff
@@ -1,9 +1,24 @@
+    borderRadius: 3px,
+    position: absolute;
+    width: 340px;
+    
```
height: 194,

Tauriqbarron replied:
```suggestion
    height: 194,
```

eliazardan committed:
Update PostsContatainer.tsx

Updated only heigh and width without absolute position or percentage width calculation

eliazardan committed:
Update PostsContatainer.tsx

Updated PostContentContainer component to replace text

Tauriqbarron reviewed (approved):
Okay there we go ! :)
